EVE.pm

```diff
@@ -0,0 +1,197 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2022] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+ EVE
+
+=head1 SYNOPSIS
+
+ cp EVE.pm ${HOME}/.vep/Plugins
+ ./vep -i variations.vcf --plugin EVE,file=/path/to/eve/data.vcf.gz
+
+=head1 DESCRIPTION
+
+ This is a plugin for the Ensembl Variant Effect Predictor (VEP) that
+ adds information from EVE (evolutionary model of variant effect).
+ In this plugin, only single nucleotide variants (SNVs) are analyzed,
```

Comment:
This would be better phrased as 'This plugin only report EVE scores for input variants and does not merge input lines to report on adjacent variants. '

Someone entering an allele string of CAC/TGG will get a score.

Reply:
Updated.